stages/org.osbuild.sfdisk

```diff
@@ -66,6 +66,10 @@ SCHEMA_2 = r"""
           "uuid": {
             "description": "UUID of the partition (GPT)",
             "type": "string"
+          },
+          "attrs": {
+            "description": "Attributes of the partition (GPT)",
+            "type": "string"
```

Comment:
We try to avoid opaque data in our options. I would make this either an array of integers, or an array of `oneOf` integers and enums, and specify the supported enums in the spec.